examples/vmck.nomad

```diff
@@ -0,0 +1,155 @@
+# TODO: remove hardcoded variabiles
+#       add configurationa like in https://github.com/liquidinvestigations/node/blob/master/templates/drone.nomad
+job "vmck" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "imghost" {
+    task "nginx" {
+      driver = "docker"
+      config {
+        image = "nginx:mainline"
+        volumes = [
+          "/usr/share/vmck-images:/usr/share/nginx/html", # TODO: add setting for host volume
+          "local/nginx.conf:/etc/nginx/nginx.conf",
+        ]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "vmck-imghost-nginx"
+        }
+      }
+      resources {
+        memory = 80
+        cpu = 200
+        network {
+          port "http" {
+            static = 10000
+          }
+        }
+      }
+      template {
+        data = <<-EOF
+          user  nginx;
+          worker_processes auto;
+
+          error_log  /var/log/nginx/error.log warn;
+          pid        /var/run/nginx.pid;
+
+          events {
+            worker_connections  1024;
+          }
+
+          http {
+            include       /etc/nginx/mime.types;
+            default_type  application/octet-stream;
+
+            log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
```

Comment:
I doubt we'll care about the logs at all, so let's just remove this.

Reply:
Question: imghost is not needed for vmck to work. Do we keep it and remove what you asked in those 2 reviews or remove the whole imghost? @mgax

Reply:
I'd say keep imghost. It's possible that, in the future, people will run a separate image hosting service, but for now, for speed, it's quite essential to have a local http server.